docs-src/references/commands.md

```diff
@@ -50,27 +50,55 @@ This Command is triggered by a call to spawn a [Child Workflow Execution](/conce
 - Awaitable: Yes, a Workflow Execution can await on the action resulting from this Command.
 - Corresponding Event: [ChildWorkflowExecutionStarted](/references/events/#childworkflowexecutionstarted)
 
+:::note
+
+OSS users cannot have more than 50000 pending Child Workflows.
```

Comment:
```suggestion
OSS users cannot have more than 50,000 pending Child Workflows.
```
Comma added per https://developers.google.com/style/numbers#commas-and-decimal-points-in-numbers and https://learn.microsoft.com/en-us/style-guide/numbers#commas-in-numbers.